hazelcast/src/main/java/com/hazelcast/config/MapConfig.java

```diff
@@ -87,6 +87,11 @@
      */
     public static final InMemoryFormat DEFAULT_IN_MEMORY_FORMAT = InMemoryFormat.BINARY;
 
+    /**
+     * We want to cache values only when an index is defined.
+     */
+    public static final CacheDeserializedValues DEFAULT_CACHED_DESERIALIZED_VALUES = CacheDeserializedValues.INDEX_ONLY;
+
```

Comment:
This would be a great piece of information for the reference-manual (I haven't looked at the PRs there, so don't know if it's there)